config_amboy.go

```diff
@@ -122,10 +122,13 @@ func (c *AmboyConfig) Set() error {
 }
 
 const (
+	// DefaultAmboyQueueName is the default name prefix for the Amboy remote
```

Comment:
Should this just say `default prefix`?

Reply:
I clarified it to say "namespace". I think if I don't state what prefix it's referring to, it might just be more confusing about what prefixes have to do with Amboy for someone unfamiliar with it.